test/unit/services/DeclarationVersionServiceSpec.scala

```diff
@@ -214,30 +219,18 @@ class DeclarationVersionServiceSpec extends UnitSpec with BeforeAndAfterEach {
         meq[UUID](correlationId.uuid).asInstanceOf[CorrelationId],
         any[ApiVersion],
         any[Option[ApiSubscriptionFieldsResponse]],
-        meq[Mrn](mrn))(any[AuthorisedRequest[_]])).thenReturn(Future.failed(new Non2xxResponseException(mockHttpResponse, 403)))
+        meq[Mrn](mrn))(any[AuthorisedRequest[_]])).thenReturn(Future.successful(Left(Non2xxResponseError(FORBIDDEN, mockHttpResponse.body))))
       val result: Either[Result, HttpResponse] = send()
 
       result shouldBe Left(ErrorResponse.ErrorPayloadForbidden.XmlResult.withConversationId)
     }
 
-    "return 404 error response when backend call fails with 404" in new SetUp() {
```

Comment:
Is this test no longer needed?

Reply:
It's not part of the spec for EIS to return HTTP 404; a 'not found' error is reflected in a HTTP 500 response with error code `CDS60001`.

The test wasn't useful to begin with.